setup.cfg

```diff
@@ -32,8 +32,9 @@ install_requires =
     cldfbench
     pyglottolog
     pyconcepticon
-    pyclts>=3.1.0
+    pyclts
     lxml
+    pyfakefs
```

Comment:
I got rid of this dependency, do we want to re-instate it? I don't think it will be that useful for us.

Reply:
Yes, I think so too. By now we have better functions to create dummy files with content.